.github/workflows/tests.yml

```diff
@@ -105,6 +105,7 @@ jobs:
             tests/logprob/test_abstract.py
             tests/logprob/test_basic.py
             tests/logprob/test_binary.py
+            tests/logprob/test_checks.py
```

Comment:
This feels very dangerous, there is no fallback for tests that are not included explicitly?

Reply:
Previously, the asserts were not being defined in this test effectively and hence the desired exceptions were not raised. I think the current changes should be sufficient to rectify this.

Reply:
The problem I am referring to is not the underlying bugs, but the fact that we were not running the tests for this file at all, which is a more broad issue.

Reply:
We have a script under `./scripts` that should have caught this. Why didn't it?

Reply:
No idea, can you take a look?

Reply:
Log indicates that the check was skipped.
Maybe because this line doesn't match the filename of the `tests.yml` workflow: https://github.com/pymc-devs/pymc/blob/4d2b2efc2b8ad7b5060d3f4a642534408b5551eb/.pre-commit-config.yaml#L67


I'm just on my phone in the train, so I can't open a PR. Can you try it on this branch?

Reply:
Opened PR in #6756